### PR TITLE
NR-00000 # modifying regex for commit-msg

### PR DESCRIPTION
**Pull request by meiao**
### Overview
Simplifying the regex for commit msg and allowing messages other than comment

### CODE REVIEW

File: dev-tools/git-hooks/hooks/commit-msg [hunk 1]
```diff
@@ -11,7 +11,7 @@ shopt -s nocasematch
 # or...if it starts with "Merge". Kept that last bit from the past impl
 #
 # Note that since our new Jira instances supports both the internal (NR) naming and the public (NEWRELIC), we test for both
-if [[ ! "$msg" =~ ^(NR-[0-9]+[ ]+\#comment|merge) ]] && [[ ! "$msg" =~ ^(NEWRELIC-[0-9]+[ ]+\#comment|merge) ]]; then
+if [[ ! "$msg" =~ ^((NR|NEWRELIC)-[0-9]+[ ]+\#.*)|merge ]]; then
   echo "Aborting commit. Your commit message must be a Jira Smart Commit, starting with ('NR-xxx #comment' or 'NEWRELIC-xxx #comment') or 'Merge'"
```
Comment (jasonjkeller) [inline dev-tools/git-hooks/hooks/commit-msg:15]:
```suggestion
  echo "Aborting commit. Your commit message must be a Jira Smart Commit, starting with ('NR-xxx' or 'NEWRELIC-xxx') or 'Merge'"
```

**Review by jasonjkeller — COMMENTED** [review]
Probably should update this comment as well.

File: dev-tools/git-hooks/hooks/commit-msg (hunk 1 above)
Comment (meiao) [reply dev-tools/git-hooks/hooks/commit-msg:15]: it still requires the `#` and some text after it.

**Review by jasonjkeller — APPROVED** [review]

**Comment by meiao** [comment]
Apparently the `#comment` is not a placeholder.

**Pull request closed without merging**